torchvision/prototype/features/_feature.py

```diff
@@ -7,7 +7,52 @@
 F = TypeVar("F", bound="_Feature")
 
 
-class _Feature(torch.Tensor):
+class _TransformsMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__()
+
+        # To avoid circular dependency between features and transforms
+        from ..transforms import functional as F
+
+        self._F = F
+
+    def horizontal_flip(self):
+        # Just output itself
+        # How dangerous to do this instead of raising an error ?
+        return self
```

Comment:
I would raise a "not implemented" error here and implement no-op explicitly on the implementations. So if bbox can't blur, then it should explicitly implement to be no-op as opposed to leaving the default implementation which returns self. Could help us avoid issues but if you disagree I'm happy to leave as-is and discuss later.

Reply:
I agree that raising `NotImplemented` could be good. The only drawback I found that all targets features like `Label`, `OneHotLabel` should also implement explicitly these no-ops. From the first glance, `OneHotLabel.blur(self, *args, **kwargs)` as no-op made me think that it is a bit too much...

Reply:
Up to you. I've added this for your consideration. Feel free to ignore or postpone for later.